src/core/qgstemporalnavigationobject.cpp

```diff
@@ -307,3 +306,19 @@ QgsTemporalNavigationObject::AnimationState QgsTemporalNavigationObject::animati
 {
   return mPlayBackMode;
 }
+
+long QgsTemporalNavigationObject::findBestFrameNumberForFrameStart( QDateTime frameStart )
+{
+  long bestFrame = 0;
+  QgsDateTimeRange testFrame = QgsDateTimeRange( frameStart, frameStart ); // socalled 'instant' Range
```

Comment:
what does "socalled" mean?

Reply:
Uh lol... that is not english :-) ? I just mean to say there that a QgsDateTimeRange in which start and finish are the same actually is called an 'Instant Range'... is 'a so callled' english?

Reply:
ah, you mean "so called"? I've never seen this written as a single word :yum: